Bottom: fdd78452d7e02d706c88b4821b9ff48bc33218cb
Top:    bd3dc75f7f730ca898630a615ebb0359ffcf66e4
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-05-03 14:59:29 -0700

test


---

diff --git a/src/cma.c b/src/cma.c
index 2fb9913..923e611 100755
--- a/src/cma.c
+++ b/src/cma.c
@@ -584,6 +584,7 @@ static int ucma_query_addr(struct rdma_cm_id *id)
 		if (ret)
 			return ret;
 		id->port_num = resp.port_num;
+printf("port num %d\n", id->port_num);
 		id->route.addr.addr.ibaddr.pkey = resp.pkey;
 	}
